server/app/blocks.go

```diff
@@ -124,6 +126,18 @@ func (a *App) DeleteBlock(c store.Container, blockID string, modifiedBy string)
 		return err
 	}
 
+	if block.Type == model.TypeImage {
+		fileName, fileIDExists := block.Fields["fileId"]
+		if fileName, fileIDIsString := fileName.(string); fileIDExists && fileIDIsString {
+			filePath := filepath.Join(block.WorkspaceID, block.RootID, fileName)
+			err = a.filesBackend.RemoveFile(filePath)
+
+			if err != nil {
+				a.logger.Error("Error deleting image file", mlog.Err(err))
```

Comment:
I recommend logging the filepath so administrators know which file they need to manually clean up.